uint16 planeID

float64 scurve

bool need_idel

bool need_take_off

bool need_land

bool need_formation

bool need_protected

uint16 formation_type
